src/main/java/dataloader/CsvDataLoader.java

```diff
@@ -0,0 +1,70 @@
+package dataloader;
+
+import org.apache.commons.csv.CSVFormat;
+import org.apache.commons.csv.CSVParser;
+import org.apache.commons.csv.CSVRecord;
+import tech.tablesaw.api.Table;
+import tech.tablesaw.io.csv.CsvReadOptions;
+
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.io.Reader;
+import java.util.TreeMap;
+
+
+/**
+ * Class that takes a String as path file
+ * to csv files
+ * and creates datasets in two formats
+ * TreeMap & Table from tablesaw.
+ */
+
+public class CsvDataLoader {
+
+    /**
+     * Simple method that parses data set from a csv file
+     * without knowing the headers of the file
+     */
+    public void parseFile(String csvFile, TreeMap dataSet) throws IOException {
+
+        Reader csvData = new FileReader(csvFile);
+
+        CSVParser parser = CSVParser.parse(csvData, CSVFormat.DEFAULT);
+
+        for (CSVRecord record : parser) {
+
+            for (String field : record) {
+                dataSet.keySet();
+                dataSet.get(field);
+
+                System.out.println("\"" + field + "\", ");
```

Comment:
Remove this ```print```. Have we tested this function with real data? Perhaps we need to have a function that accepts a ```File``` argument just like the case with ```Table```.

Reply:
I am not sure about the last point. will remove the the print statement in any case & make some test runs with dummy data.